src/style.css

```diff
@@ -1 +1,112 @@
-/* add styles here */
+html {
+  --main-color: var(--main-color);
+}
+
+@font-face {
```

Comment:
better:
```
@font-face {
  font-family: Avenir;
  src: url('../../fonts/Avenir-Heavy.ttf');
  src:
    local('../../fonts/Avenir-Heavy.ttf'),
    url('../../fonts/Avenir-Heavy.ttf') format('truetype');
  font-weight: 700;
  font-style: normal;
}
```

and move on all fonts to fonts.css and include here @import './fonts.css';